### DISCUSSION TITLE
Code markup needs some love in 3.0 beta

### DISCUSSION THREAD

**Discussion by erutan** [Zettlr/Zettlr#4410] [Q&A]
Using the latest 3.0 beta and I'm really liking it so far, but code is bugging me. 

Looking at https://docs.zettlr.com/en/reference/keyboard-shortcuts/ there doesn't seem to be any keyboard shortcuts for inline code  `foobar` with one tick on each side or code blocks with three ticks on each side. At the very least having a shortcut for inline code, then a proper 3 tick block code popup in the popup would be nice.

The pop-up formatter has icons, but they're not as fast to use and are just two different ways of doing inline comments (the first of which is a HTML comment and not one I'd use with markdown, the block code just does inline code.

Content within inline code ticks is styled different typographically, but it'd be nice to also have the standard gray background color to it, though perhaps that's something I can do forking a theme. :)

**Comment by exanrauzer** [comment]
I agree with you in general, and I think the issue of styling is likely left towards the end after addressing the bugs that would enable the release of version 3.0. Prioritizing bug fixes and stability before focusing on cosmetic enhancements is a common approach in software development. Once the core functionality is solid, developers can then fine-tune the visual aspects and customization options.

**Comment by erutan** [comment]
Yeah I wouldn’t mark this as a critical or blocking issue, but it’s something that should be addressed. Just having a keyboard shortcut for it would be great ala Discourse, slack, discord, etc. 

While this is definite scope creeping, it’d be interesting to be able to create custom shortcuts - say:

* wrap selection function
* backticks ‘ are wrappers
* Cmd-E is shortcut.
